content/docs/user_guide/debugging.md

```diff
@@ -89,7 +89,7 @@ the options available:
 For example:
 
 ```bash
-docker run --runtime=runsc-prof --rm -d alpine sleep 1000
+docker run --runtime=runsc-prof --rm -d alpine sh -c "while true; do echo running; sleep .1; done"
```

Comment:
Printing "running" ~10 times a second seems a bit excessive. Maybe sleeping a second is a better?